packages/mask/src/extension/popups/components/BottomDrawer/index.tsx

```diff
@@ -35,6 +35,10 @@ export interface BottomDrawerProps extends PropsWithChildren {
 export const BottomDrawer = memo<BottomDrawerProps>(function BottomDrawer({ open, onClose, children, title }) {
     const { classes } = useStyles()
     const handleClose = () => onClose?.()
+    const everOpenRef = useRef(false)
+    useEffect(() => {
```

Comment:
Why not?

if !open return false

Reply:
> if !open return false

That will disable transition during opening and closing. 

While `{open ? children : null}` will make content blank during closing.